src/helpers/tokenTypes.js

```diff
@@ -1,9 +1,3 @@
-const WARD_STAFF = "wardStaff";
-const TRUST_ADMIN = "trustAdmin";
-const ADMIN = "admin";
-
-module.exports = {
-  WARD_STAFF,
-  TRUST_ADMIN,
-  ADMIN,
-};
+export const WARD_STAFF = "wardStaff";
```

Comment:
Is it time to rename this helper to `userTypes` ?

Reply:
That's a good shout!

Reply:
Updated in https://github.com/madetech/nhs-virtual-visit/pull/382/commits/0b1a5fdb6f470305b76cd4e1e7c15de1da6bcdd2.